mne/viz/_3d.py

```diff
@@ -990,6 +1078,225 @@ def time_label(t):
+
+    Note
+    ----
+    PySurfer currently needs the SUBJECTS_DIR environment variable, which will
+    automatically be set by this function. Plotting multiple SourceEstimates
+    with different values for subjects_dir will cause PySurfer to use the wrong
+    FreeSurfer surfaces when using methods of the returned Brain object. It is
+    therefore recommended to set the SUBJECTS_DIR environment variable or
+    always use the same value for subjects_dir (within the same Python
+    session).
+    """
+    import surfer
+    from surfer import Brain
+    import mayavi
+
+    # import here to avoid circular import problem
+    from ..source_estimate import VectorSourceEstimate
+
+    surfer_version = LooseVersion(surfer.__version__)
+    v06 = LooseVersion('0.6')
+    if surfer_version < v06:
+        raise ImportError("This function requires PySurfer 0.6 (you are "
+                          "running version %s). You can update PySurfer "
+                          "using:\n\n    $ pip install -U pysurfer" %
+                          surfer.__version__)
+
+    if not isinstance(stc, VectorSourceEstimate):
+        raise ValueError('stc has to be a vector source estimate')
+
+    if hemi not in ['lh', 'rh', 'split', 'both']:
+        raise ValueError('hemi has to be either "lh", "rh", "split", '
+                         'or "both"')
+
+    # check `figure` parameter (This will be performed by PySurfer > 0.6)
+    if figure is not None:
+        if isinstance(figure, int):
+            # use figure with specified id
+            size_ = size if isinstance(size, (tuple, list)) else (size, size)
+            figure = [mayavi.mlab.figure(figure, size=size_)]
+        elif not isinstance(figure, (list, tuple)):
+            figure = [figure]
+        if not all(isinstance(f, mayavi.core.scene.Scene) for f in figure):
+            raise TypeError('figure must be a mayavi scene or list of scenes')
+
+    subjects_dir = get_subjects_dir(subjects_dir=subjects_dir,
+                                    raise_error=True)
+    subject = _check_subject(stc.subject, subject, True)
+    if hemi in ['both', 'split']:
+        hemis = ['lh', 'rh']
+    else:
+        hemis = [hemi]
+
+    title = subject if len(hemis) > 1 else '%s - %s' % (subject, hemis[0])
+
+    # Plot the surface meshes
+    with warnings.catch_warnings(record=True):  # traits warnings
+        brain = Brain(subject, hemi=hemi, surf=surface, title=title,
+                      curv=False, size=size, background=background,
+                      foreground=foreground, figure=figure,
+                      subjects_dir=subjects_dir, views=views)
+
+        # Set the transparancy of the meshes
+        for row in brain._figures:
+            for fig in row:
+                for child in fig.children:
+                    props = child.children[0].children[0].actor.property
+                    props.trait_set(opacity=brain_alpha, backface_culling=True)
```

Comment:
oof... can you open a PR to PySurfer to fix backface culling if it's not already set in 0.7?